src/dune/install_rules.ml

```diff
@@ -170,17 +174,37 @@ end = struct
                let dune_package_file =
                  Package_paths.dune_package_file ctx pkg
                in
+               let toplevel_lib_meta_files =
```

Comment:
It seems to me that we could use `pkg.deprecatd_package_names` here

Reply:
We could, but right now we don't check that every name mentioned in `(deprecated_package_names)` actually appears in a `deprecated_library_name` stanza. Should we enforce that?

Reply:
This is similar to checking that packages are empty, which we don't do, so we shouldn't do it here either.